t/nqp/019-file-ops.t

```diff
@@ -1,4 +1,4 @@
-#! nqp
+#!/usr/bin/env nqp
```

Comment:
We may as well remove this line. Since not even all the test files have a shebang line. (Also the files don't have the execute bit either).

Reply:
done.

I'll appreciate it if you can test this change on a macOS, IIRC the last time @coke uncomment this test, it causes regression. that's why i don't merge this pr yet.